streams/src/main/java/org/apache/kafka/streams/state/internals/metrics/StateStoreMetrics.java

```diff
@@ -146,13 +144,6 @@ private StateStoreMetrics() {}
     private static final String SUPPRESSION_BUFFER_SIZE_MAX_DESCRIPTION =
         MAX_DESCRIPTION_PREFIX + SUPPRESSION_BUFFER_SIZE_DESCRIPTION;
 
-    private static final String EXPIRED_WINDOW_RECORD_DROP = "expired-window-record-drop";
```

Comment:
This metric is removed as part of KIP-743, and it's only used in tests (which I also cleaned up as a piggy-back).